bx_py_utils/graphql_introspection.py

```diff
@@ -51,7 +51,8 @@ def _resolve_type(types_by_name, graphql_type):
         return graphql_type
 
 
-def _craft_complete_query(types_by_name, visited_types, graphql_type, allow_recursive, indent=''):
+def _craft_complete_query(types_by_name, visited_types, graphql_type, allow_recursive, max_depth,
```

Comment:
why not with default, e.g.: max_depth=None

Reply:
This is an internal method and not the public interface. But sure, why not? Added a default argument.